src/components/AddItem.js

```diff
@@ -2,6 +2,16 @@ import React, { useState } from 'react';
 import { useHistory } from 'react-router-dom';
 import { db } from '../lib/firebase';
 import Error from './Error';
+import TextField from '@material-ui/core/TextField';
```

Comment:
Nit: I'm guessing that in the @material-ui package, that these components are likely all exported from an index.js and you can probably clean up the imports a little by doing them like:

import { Radio, RadioGroup, FormControlLabel, ... } from "@material-ui/core";

Reply:
Thanks Mike. I thought that list of imports was getting out of control!

Reply:
More imports !!!!!!!